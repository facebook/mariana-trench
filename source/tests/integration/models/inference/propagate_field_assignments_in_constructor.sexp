(method (private) "LFlow;.<init>:(LData;LData;)V"
 (
  (load-param v0)
  (load-param v1)

  LINE(1)
  (iput-object v1 v0 "LFlow;.field:LData;")

  (return-void)
 )
)
