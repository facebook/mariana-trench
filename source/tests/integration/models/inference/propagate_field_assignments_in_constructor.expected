{
  "metadata" :
  {
    "@generated" : true
  },
  "models" :
  [
    {
      "method" : "LFlow;.<init>:(LData;LData;)V",
      "position" :
      {
        "line" : 0,
        "path" : "Flow.java"
      },
      "propagation" :
      [
        {
          "input" : "Argument(1)",
          "output" : "Argument(0).field"
        }
      ]
    }
  ]
}
